### ISSUE TITLE
[Accessibility], recommended to fix: Using the concept lists with the keyboard is heavy and unusable.

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1982]
## Description of the problem
Using the concept lists of the vocabularies with the keyboard is heavy and unusable. All links in the sidebar (1710 links in Hierarchy view) are tab-focusable. If the keyboard user would like to move the focus to the main content (to the data of the selected term), the keyboard user must first move through all of the nearly 2000 links (or collapse them) before the focus moves to the main content.
This is not a WCAG violation but affect significantly to keyboard usage and is therefore recommended to fix.

**Fix**: 
**Option 1**: Keyboard use would be much easier if activating a tree item moves focus to the beginning of the new content, ideally a level one heading with content that matches the name of the tree item that was activated. Focusing on the heading informs screen reader users that navigation is complete and confirms the destination. when pressing shif+ tab, focus returns where it was on treeview. Also in this case Keyboard interaction is recommended to follow the ARIA tree pattern: Up/Down Arrow moves between visible items, Right Arrow expands a collapsed item, Left Arrow collapses an expanded item, Enter or Space selects the current item, and Home/End moves to the first or last item. 
example about similar implementation. https://dequeuniversity.com/library/aria/tree-view 

**Option 2**: Other option is that the focus stays at the treeitem when the treeitem is activated. The treeitems could only be browsed with arrow-keys (up and down arrows), and when pressing the tab- button  the focus moves to the main content.

## Additional information (e.g. screenshots) about the problem

<img width="1004" height="424" alt="Image" src="https://github.com/user-attachments/assets/33ea0994-e2bd-4679-94dd-c6a8435579a6" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
<img width="583" height="410" alt="Image" src="https://github.com/user-attachments/assets/9b6b886d-128d-4afd-916f-1e70b7a143ce" />

<img width="562" height="338" alt="Image" src="https://github.com/user-attachments/assets/4b5ab9ff-fec7-47a8-a03b-470158aa9284" />

There is still an issue with the tree. I made a new ticket about this and linked this.
